src/styles/main.scss

```diff
@@ -20,6 +20,7 @@ body {
   text-align: center;
   vertical-align: center;
   user-select: none;
+  transition: all 0.5s;
```

Comment:
Ensure that the SCSS file is compiled to CSS before linking it in the HTML file, as browsers do not support SCSS directly.